### ISSUE TITLE
PassFF filling login form when pass store doesn't contain entry

### ISSUE THREAD

**Issue by czardien** [passff/passff#456]
* Versions
        Operating system: Archlinux, Linux 5.8.12
        Browser: Firefox 81.0
        PassFF: 1.10.3
        Host app: 1.2.1-3

Status line output: [hh:mm:ss] show -> (0)

[17:14:50] show -> (0) no error message

### Actual behaviour

I just installed the **PassFF** extension, along with **pass** and the **pass-host** application. My password store contains a single entry for website: `foo.com/login`. 

Visiting a previous site `bar.com/login` for which I had a login set in Firefox's own login store I got an autofill which was stamped with PassFF logo.

I find this confusing as this is implying that the login and password is in my pass store, when it's not. 

### Expected behaviour

No autofill happens when the password is not in the store.

**Comment by tuxor1337** [comment]
There is no login form on bar.com, nor on foo.com. Can't reproduce. Please give step-by-step instructions on how to reproduce this.

**Comment by czardien** [comment]
Hi @tuxor1337, sorry for the late reply. There is no login form on bar.com and foo.com. I agree this was a very confusing way for me to ask a question so let me rephrase and please bear with me.

How does Firefox native password store and **pass** work together? Is there such a mechanism in place? For instance, is there a way to completely bypass Firefox's login handling mechanism and only store new login with **passff** for instance?  

The reason I am asking is because I am now finding myself regularly updating my **pass** store manually when I login to new websites from Firefox, which is a bit of a pain, and I can't really find in the docs if I am missing something.

The scenario I was referring to was about me being confused when I had Firefox auto-filling a login form, stamped with the **passff** logo, which led me to believe the login form was in my **pass-store** when my **pass-store** didn't contain this entry. 

Hope that is clearer!

**Comment by tuxor1337** [comment]
Thanks for rephrasing!

As far as I know, extensions can't interfere with Firefox's built-in password manager in any way whatsoever. The extension doesn't know whether you are using the built-in password manager nor does it know whether you have any passwords stored in it.

PassFF is not a replacement for the built-in password manager. From the README: "This extension will allow you to access your zx2c4 pass repository directly from your web browser." Everything else is candy.

However, you might be interested in these feature requests: #380 #418  I'm closing this issue as a duplicate of those.
